docs/whats-new.md

```diff
@@ -2,6 +2,35 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.8
+
+Release date: June, 2022
+
+### TypeScript
+
+Official typings are now available in deck.gl for public preview. It was a tremendous effort converting the deck.gl code base into TypeScript. We did it not just for TypeScript users to easily consume this library, but also for the overall robustness and maintainability of our own code base.
+
+In v8.x, the types will not be exposed by default to prevent any typing errors from breaking existing TypeScript applications. You can opt in to using the typed version by switching your import statements to the `@deck.gl/*/typed` endpoints. Visit [getting started with TypeScript](/docs/get-started/using-with-typescript.md) to find out more.
```

Comment:
Do we want to give an indication as to when we plan on exposing them by default? To give advance warning and to encourage people to try them